.github/push-image.sh

```diff
@@ -0,0 +1,29 @@
+#!/bin/bash
```

Comment:
compare https://github.com/libero/scripts/blob/master/travis/push-image.sh

Reply:
We should move this there (`libero/script`).

If it needs testing, let's do it on a dummy repo.

Reply:
It should ideally tested in the `script` repository itself, to validate PRs.

Reply:
Working on https://github.com/libero/scripts/pull/9 to eventually do that